app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, dependent: :destroy
+  has_many :comments, through: :posts, dependent: :destroy
+
+  validates :username,
+            presence: true,
+            length: { minimum: 6, maximum: 20 },
+            uniqueness: true
```

Comment:
### [OPTIONAL]
A name can be less than 6 characters. Example "peter" so I don't think the minimum of 6 is actually required